packages/integration/src/github/config.ts

```diff
@@ -93,6 +93,10 @@ export type GithubAppConfig = {
    * Client secrets can be generated at https://github.com/organizations/$org/settings/apps/$AppName
    */
   clientSecret: string;
+  /**
+   * List of installation owners allowed to be used by this backstage https://github.com/app/installations/$InstallationId
```

Comment:
This leads to a 404

Reply:
The link was wrong. I also added a bit of detail in this code comment. Specifically, the App installation list can only retrieved by the Application via its login token signed by it's private key. I pointed at the relevant docs.

Let me know how it reads. Happy to review again if needed. :)